samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -340,6 +340,25 @@ See :ref:`ug_bt_fast_pair_provisioning` for details.
    You cannot use the sample without the Fast Pair provisioning data.
    Programming the device with the sample firmware without providing the proper Fast Pair provisioning data results in assertion failure during boot.
 
+Release build
+=============
+
+You can build the sample with a release configuration to demonstrate how to adapt the default sample configuration for production, using the ``-DFILE_SUFFIX=release`` flag in your build.
```

Comment:
Should we state `to adapt the default sample configuration for production`? Maybe we should only inform that it disables assertions+logs, enables LTO and it optimizes memory consumption? Keep in mind that we do not even support bootloader here, so it's still a sample - not reference design